azure-pipelines.yml

```diff
@@ -21,5 +40,8 @@ jobs:
         displayName: 'Build'
         condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/heads/master'))
         inputs:
+          templatePath: $(templatePath)
           command: build
           azureSubscription: dts-management-prod-intsvc
+          variables: |
+              azure_image_version=$(get_azure_image_version.azure_image_version)
```

Comment:
I dont think this is simplified - lets catchup

Reply:
Morning Thomas - sure - available when you are..

Reply:
need that morning coffee, it looks fine on second look